sites/language/includes/languagetoggle.html

```diff
@@ -4,7 +4,7 @@ <h2 class="wb-inv">{{ i18nText-language }}</h2>
 		<li>
 			<a lang="{{ i18nText-altLang }}" hreflang="{{ i18nText-altLang }}" href="{{ page.altLangPage }}">
 				<span class="hidden-xs">{{ i18nText-altLanguage }}</span>
```

Comment:
This one should also have the attribute:
```suggestion
				<span class="hidden-xs" translate="no">{{ i18nText-altLanguage }}</span>
```